pkg/Library/LocalStorage/Atoms/LocalStorage.js

```diff
@@ -12,6 +12,7 @@ shouldUpdate({key}) {
   return key;
 },
 async update({key, storeValue}, state, {service}) {
+  //key = `${globalThis.config.aeon}/${key}`;
```

Comment:
maybe there should be a `keyPrefix` input instead, so this can be controlled by the graph?